include/experimental/__p0009_bits/layout_stride.hpp

```diff
@@ -51,12 +51,21 @@ namespace detail {
     std::is_same<typename Layout::template mapping<typename Mapping::extents_type>, Mapping>::value;
 
 #if defined(_MDSPAN_USE_CONCEPTS) && MDSPAN_HAS_CXX_20
+  namespace internal {
+  namespace detail {
+  template <typename _Tp, typename _Up>
+  concept __same_as = std::is_same_v<_Tp, _Up>;
+  } // namespace detail
+  template <class T, class U>
+  concept __same_as = detail::__same_as<T, U> && detail::__same_as<U, T>;
+  } // namespace internal
```

Comment:
Would you consider only using the work-around if the `<concepts>` header is not available?  It's nicer to have the same symbol in the common case of "the compiler actually implements C++20 correctly."

```suggestion
#if defined(_MDSPAN_USE_CONCEPTS) && MDSPAN_HAS_CXX_20
#  if ! defined(__cpp_lib_concepts)
  namespace internal {
  namespace detail {
  template <typename _Tp, typename _Up>
  concept __same_as = std::is_same_v<_Tp, _Up>;
  } // namespace detail
  template <class T, class U>
  concept __same_as = detail::__same_as<T, U> && detail::__same_as<U, T>;
  } // namespace internal
#  endif
```

Reply:
I normally recommend against this sort of macro-heavy C++ code, so please consider this and the change below optional.  I don't think "concept aliases" are a thing (by which I mean, "if `<concepts>` exists, alias `__same_as` to `std::same_as`, else define it like this").